python/lib/sift_py/ingestion/config/telemetry.py

```diff
@@ -179,6 +188,25 @@ def _from_yaml(
             )
 
         for rule in config_as_yaml.get("rules", []):
+            namespace = rule.get("namespace")
+
+            if namespace:
+                rule_namespace = rule_namespaces.get(str(namespace))
+                if not rule_namespace:
+                    # TODO: This is a placeholder.
```

Comment:
A placeholder because if this pattern makes sense we can add a more specific and helpful error.